Founding-membership.md

```diff
@@ -0,0 +1,67 @@
+# Become a founding member
+
+We’re currently looking for founding members to help build the Foundation for Public Code’s long term future by joining us now.
+
+Founding members are forward-thinking public organisations with a strong principled commitment to the existence of an ecosystem of public code. You’ll be willing to take co-ownership of the Foundation [in line with our draft principles] and be an advocate for public code.
+
+We are looking for organisations that can either:
+
+* help us validate our code certification and stewardship processes
+* provide material support to create a public code ecosystem
+
+We’ll open other types of membership from 2020.
+
+## Benefits of being a founding member
+
+As a founding member, you’ll shape the Foundation’s processes and governance.
+
+As a founding member, you will:
+
+* be one of the first to submit your codebases for stewardship by the Foundation (reducing your future risk and maintenance costs)
+* get a seat on the board of directors
+* have your membership fees waived for the first year
+* steer the Foundation’s pipeline towards projects you’d like to reuse
+* gain recognition as a leader of systemic digital transformation
+
+This is in addition to the regular benefits of Foundation membership. These include being able to:
+
+* invoke continuous integration auditing resources on projects in development accepted by the Foundation (increasing the security of your code)
+* (re)use codebases from the repository, and rely on the stewarded community for support, bug fixes, and continued development (saving money on future procurement)
```

Comment:
Anyone gets this benefit, perhaps we can put something in there about there privileged role